solidity/contracts/lib/verifier_anon.sol

```diff
@@ -68,7 +68,7 @@ contract Groth16Verifier_Anon {
     function verifyProof(uint[2] calldata _pA, uint[2][2] calldata _pB, uint[2] calldata _pC, uint[4] calldata _pubSignals) public view returns (bool) {
         assembly {
             function checkField(v) {
-                if iszero(lt(v, q)) {
+                if iszero(lt(v, r)) {
```

Comment:
Not sure what the context for the list of changes is here but this is generate and I don't see a circuit change in the PR

Reply:
it's due to the different versions of the snarkjs tool when generating the solidity verifier code